src/main/java/modal/StringAddCalculator.java

```diff
@@ -0,0 +1,59 @@
+package modal;
+
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static String DELIMITER = "[,:]";
+    private static final String CUSTOM_DELIMITER_REGEX = "//(.)\\n(.*)";
+
+    public static int splitAndSum(String inputString) {
+        if(checkNullinput(inputString)) {
```

Comment:
자바 컨벤션이 지켜지지 않은 부분이 있네요!
```suggestion
        if (checkNullinput(inputString)) {
```
아래 링크의 7.4절을 참고하시면 좋을 것 같아요 :)
https://myeonguni.tistory.com/1596